_posts/2016-11-23-Support-for-MMTF.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: Support for MMTF has arrived!
+---
+
+The upcoming 0.16.0 release of MDAnalysis will have
+[support for MMTF](https://twitter.com/mmtf_spec/status/799704395046760448)!
+[MMTF](http://mmtf.rcsb.org/) is a new format designed to provide compact, efficient and fast browsing of the [Protein
+Data Bank](www.rcsb.org/).
+Support for MMTF within MDAnalysis is offered through reading locally stored MMTF files,
+or through fetching the file directly from the PDB archive through the new
+`MDAnalysis.fetch_mmtf` function.
```

Comment:
Maybe you can add that the mmtf files are very small and will be quick to download also on slow connections.